### PR TITLE
#570 maven pom.xml plugins versions update

### PR DESCRIPTION
**Pull request by flslkxtc**

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#596](https://github.com/yegor256/cactoos/issues/596) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#596](https://github.com/yegor256/cactoos/issues/596) is assigned to @neonailol[/z](https://www.0crat.com/u/neonailol). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: pom.xml [hunk 1]
```diff
@@ -81,6 +81,9 @@ The MIT License (MIT)
       <url>https://github.com/yegor256/cactoos</url>
     </site>
   </distributionManagement>
+  <prerequisites>
+    <maven>3.0.4</maven>
```
Comment (neonailol) [inline pom.xml]: this is not correct, prerequisites section are only for maven plugin projects, to define required maven version, we need to specify it via maven enforcer plugin in the build section

Comment (flslkxtc) [reply pom.xml]: Thanks, i've fixed it

**Comment by neonailol** [comment]
@rultor good to merge

**Comment by neonailol** [comment]
@yegor256 please take a look at this

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @neonailol[/z](https://www.0crat.com/u/neonailol), total is [+510](https://www.0crat.com/u/neonailol)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+9565](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#596](https://github.com/yegor256/cactoos/issues/596) is now out of scope

**Pull request merged**